docs/tutorials/coord_sys_tutorial.md

```diff
@@ -0,0 +1,211 @@
+
+See the code [here](https://github.com/open-mmlab/mmdetection3d/blob/master/mmdet3d/core/bbox/structures/cam_box3d.py) for more details.
+
+### Rotation of boxes
+
+We set the rotation of all kinds of boxes to be counter-clockwise about the gravity axis. Therefore, to rotate a 3D box we first calculate the new box center, and then we add the rotation angle to the yaw angle.
+
+See the code [here](https://github.com/open-mmlab/mmdetection3d/blob/master/mmdet3d/core/bbox/structures/cam_box3d.py) for more details.
+
+## Common FAQ
+
+#### Q1: Are the box related ops universal to all coordinate system types?
+
+No. For example, the ops under [this folder](https://github.com/open-mmlab/mmdetection3d/blob/master/mmdet3d/ops/roiaware_pool3d) are applicable to boxes under Depth or LiDAR coordinate system only. The evaluation functions for KITTI dataset [here](https://github.com/open-mmlab/mmdetection3d/blob/master/mmdet3d/core/evaluation/kitti_utils) are only applicable to boxes under Camera coordinate system since the rotation is clockwise if viewed from above.
+
+For each box related op, we have marked the type of boxes to which we can apply the op.
+
+#### Q2: In every coordinate system, do the three axes point exactly to the right, the front, and the ground, respectively?
+
+No. For example, in KITTI, we need a calibration matrix when converting from Camera coordinate system to LiDAR coordinate system.
+
+#### Q3: How does a phase difference of <img src="https://render.githubusercontent.com/render/math?math=2\pi"> in the yaw angle of a box affect evaluation?
+
+For IoU calculation, a phase difference of <img src="https://render.githubusercontent.com/render/math?math=2\pi"> in the yaw angle will result in the same box, thus not affecting evaluation.
+
+For angle prediction evaluation such as the NDS metric in NuScenes and the AoS metric in KITTI, the angle of predicted boxes will be first standardized, so the phase difference will not change the result.
+
+#### Q4: How does a phase difference of <img src="https://render.githubusercontent.com/render/math?math=\pi"> in the yaw angle of a box affect evaluation?
+
+For IoU calculation, a phase difference of <img src="https://render.githubusercontent.com/render/math?math=\pi"> in the yaw angle will result in the same box, thus not affecting evaluation.
+
+However, for angle prediction evaluation, this will result in the exact opposite direction.
+
+Just think about a car. The yaw angle is the angle between the direction of the car front and the positive direction of the x-axis. If we add <img src="https://render.githubusercontent.com/render/math?math=\pi"> to this angle, the car front becomes the car rear.
+
+For categories such as barrier, the front and the rear have no difference, therefore a phase difference of <img src="https://render.githubusercontent.com/render/math?math=\pi"> will not affect the angle prediction score.
```

Comment:
Do we need to run the `tools/create_data.py` again if we upgrade new coordinate systems?

Reply:
Yes, or use the scripts in #774